.travis.yml

```diff
@@ -14,15 +16,19 @@ branches:
 
 stages:
   - 'Lint markdown files'
-  - 'Test'
-  - 'Build'
-
+  - 'Tests'
+  - 'Trigger FSC Tests'
+  - 'Test Build using latest tag (no upload)'
+  - 'Build, Upload and Publish (draft)'
```

Comment:
Is this equal to prerelease? If so, should it be called Publish (prerelease)?

Reply:
Nevermind, I see https://developer.github.com/v3/repos/releases/#create-a-release